README.md

```diff
@@ -0,0 +1,5 @@
+# nesta_ds_utils
+
+This repo contains a python package with basic data science utility functions to assist the DAP team. 
+
+### This package is currently under development
```

Comment:
Doesn't have to be this PR, but could be worth adding some contributor guidelines and some info on requesting/suggesting new features.

Reply:
Yes, that's a good point. I will open a new issue for that that can be addressed in a different PR, at least on the meantime we  have the basic structure there.